src/main/java/com/gmail/inverseconduit/commands/sets/CoreBotCommands.java

```diff
@@ -182,6 +186,9 @@ private void createNumberCommand() {
     private void createShutdownCommand(ChatInterface chatInterface) {
         CommandHandle shutdown =
                 new CommandHandle.Builder("shutdown", message -> {
+                	if (!Permissions.isAdmin((long) message.getUserId(), BOT_CONFIG)) {
```

Comment:
static import could make this a bit nicer-looking, but that seems like personal preference ... @Unihedro what's our opinion on static imports?